File: beecrowd/1020/1020 - Age in Days .cpp
#include <iostream>

int main()
{
	int years, months, days;

	std::cin >> days;

	years = days / 365;
	days = days % 365;
	months = days / 30;
	days = days % 30;

	std::cout << years << " ano(s)" << std::endl;
	std::cout << months << " mes(es)" << std::endl;
	std::cout << days << " dia(s)" << std::endl;

	return 0;
}

File: beecrowd/1020/1020 - Age in Days .cs
using System;

class Program
{
	static void Main(string[] args)
	{
		int years, months, days;
		
		days = int.Parse(Console.ReadLine());
		
		years = days / 365;
		days = days % 365;
		months = days / 30;
		days = days % 30;
			
		Console.WriteLine(years + " ano(s)");
		Console.WriteLine(months + " mes(es)");
		Console.WriteLine(days + " dia(s)");
	}
}